pocket/models.py

```diff
@@ -23,16 +33,28 @@ class User(models.Model):
                                     {PAYMENT_OFF, '미결제'}
                                   ]
     
-    payment_status              = models.IntegerField(choices=PAYMENT_CHOICES, default=PAYMENT_OFF, verbose_name='결제상태')
+    payment_status              = models.IntegerField(choices=PAYMENT_CHOICES, default=PAYMENT_OFF, verbose_name='결제상태', null=True)
     created_at                  = models.DateTimeField(verbose_name='생성일', auto_now_add=True)
     updated_at                  = models.DateTimeField(verbose_name='갱신일', auto_now=True)
 
+    # User 모델의 필수 field
+    is_active                   = models.BooleanField(default=True)    
+    is_admin                    = models.BooleanField(default=False)
```

Comment:
이거는 어디에 쓰이는 건가요?

Reply:
is_active, is_admin 필드는 장고(django)의 유저 모델(User Model)의 필수 필드입니다.

is_active : 활성화 여부 (기본값은 True)
is_admin : 관리자 권한 여부 (기본값은 False)

Reply:
- is_active 상세 설명.
    - Django admin 페이지에서 확인된 Active의 의미
        - Designates whether this user should be treated as active. 
           Unselect this instead of deleting accounts. 
        - 즉 user를 hard delete하지 말고 이 속성을 사용해서 soft delete하란 의미입니다.

- is_anonymous / is_authenticated / is_active
    - is_anonymous는 로그아웃 여부를 묻습니다. 로그아웃 시 True.
    - is_authenticated는 로그인 여부를 묻는 것입니다. 로그인 시 True. 
       is_anonymous와 반대된다고 생각하시면 됩니다.
     - is_active는 단순히 로그인, 로그아웃을 넘어서 그 유저의 현재 활동 가능 상태를 판단합니다. is_authenticated의 조금 더 발전된 버전입니다.